notebooks/COS/ViewData/ViewData.ipynb

```diff
@@ -26,9 +26,9 @@
     "    \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.    ax0.set_ylabel(r'Flux [$erg\ s^{-1}\ cm^{-2}\ Angstrom^{-1}$]',
</pre><p>The invalid escape sequence error in the code can be fixed by prefixing the string with the letter "r".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>